NW6 | Pedro Eugenio | Eslint and Prop-types section (#40)
CodeYourFuture/Module-React: from-scratch/README.md

You know how sometimes you get code review telling you to remove files like /node_modules or .vscode settings? Vite has created a `.gitignore` file for you. Have a look at it and see if you agree with what it's ignoring. You can add more files to ignore if you want.

## Eslint and Prop-types

> New Vite projects may encounter issues with Eslint, displaying red marks below some variables. This occurs because the default Eslint configuration requires prop-types in React projects. Here's how to resolve it:

Reviewer: ```suggestion
Eslint might display red marks below some variables in a new Vite project. This happens because the default Eslint configuration requires prop-types in React projects. Here's how to resolve it:
```

> 1. Locate the file named `.eslintrc.cjs` in your project's root folder.

Reviewer: ```suggestion
1. Find the file `.eslintrc.cjs` in your project's root folder.
```
